src/UI/UsdValue/index.tsx

```diff
@@ -9,7 +10,10 @@ const UsdValue = (props: {
   const { amount, usd, ...dataTestId } = props;
   const value = `≈ $${getUsdValue({ amount, usd })}`;
   return (
-    <small className='form-text text-secondary mt-0' {...dataTestId}>
+    <small
+      className={`${globalStyles.formText} ${globalStyles.textSecondary} ${globalStyles.mt0}`}
+      {...dataTestId}
```

Comment:
is this correct?

Reply:
The logic was not changed. Only the `className` property was touched.